packages/core/src/browser/core-preferences.ts

```diff
@@ -80,6 +80,11 @@ export const corePreferenceSchema: PreferenceSchema = {
             default: 'code',
             markdownDescription: nls.localizeByDefault('Controls the dispatching logic for key presses to use either `code` (recommended) or `keyCode`.')
         },
+        'window.extendedTabBarPreview': {
```

Comment:
I'm trying to think of what the best name would be here, what do we mean by `extended`?
Perhaps something like the following would be better/clearer:

- `window.tabbar.hoverPreview`
- `window.tabbar.enhancedHover`
- `window.tabbar.enhancedPreview`

Reply:
Good point! I went with `window.tabbar.enhancedPreview` and updated the PR description as well.